src/Common/src/CoreLib/System/IO/Path.Windows.cs

```diff
@@ -5,7 +5,14 @@
 using System.Diagnostics;
 using System.Text;
 
+#if MS_IO_REDIST
```

Comment:
The ifdefs may look better if they are just around the single namespace line. The extra usings do not need to be under ifdef.

Reply:
Ok, I'll try and collapse them in